sdk/communication/communication-phone-numbers/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### Other Changes
 
+- Update minimum version of dependency "@azure/core-client" to "1.5.0"
```

Comment:
Could you please add additional context as to why this is required? Something like the following should do it:

```suggestion
- Update minimum version of dependency "@azure/core-client" to "1.5.0", which allows adding additional policies to the rest pipeline after initialization.
```

Reply:
sure